assets/pages/new-contributions-landing/components/ContributionAmount.jsx

```diff
@@ -118,7 +118,7 @@ function ContributionAmount(props: PropTypes) {
           icon={iconForCountryGroup(props.countryGroupId)}
           onInput={e => props.updateOtherAmount((e.target: any).value)}
           isValid={props.checkOtherAmount(props.otherAmount || '')}
-          checkoutFormHasBeenSubmitted={props.checkoutFormHasBeenSubmitted}
+          formHasBeenSubmitted={props.checkoutFormHasBeenSubmitted}
```

Comment:
why did you decide to have different names for prop and attribute?

Reply:
because different forms have `ContributionTextInput`s, sometimes the checkout form, sometimes the password set form